core/notifications/locales/es.yml

```diff
@@ -58,8 +58,8 @@ transaction.paid_invoice.title: "%{walletCurrency} Transacción"
 transaction.paid_invoice.body: "+%{formattedCurrencyAmount}"
 transaction.paid_invoice.body_display_currency: "+%{displayCurrencyAmount} | %{formattedCurrencyAmount}"
 
-price_changed.title: "Bitcoin está en movimento!"
-price_changed.body: "Bitcoin ha augmentado %{percent_increase}% en el ultimo dia %{price}!"
+price_changed.up.title: "Bitcoin está en movimento!"
+price_changed.up.body: "Bitcoin ha augmentado %{change_percent}% en el ultimo dia %{price}!"
```

Comment:
Why are we not using transifex for notifications?

Reply:
We do use transifex. I think we can simply add the Spanish translation in this pr itself. Even if we do not then it will default to English.

Reply:
Decided that it can come in via transifex only.